SE-1186 Updates the documentation to use the updated ansible_single playbook

pomegranited opened the pull request:
Updates our analytics deployment documentation to use the upstream `ansible_single.yml` playbook instead of resources in this repo.

**JIRA tickets**: SE-1186

**Dependencies**: https://github.com/edx/configuration/pull/5223

**Testing instructions**:

See https://github.com/edx/configuration/pull/5223

**Reviewers**
- [x] @gr4yscale

pomegranited committed:
Updates the documentation to use the ansible_single playbook

updated by https://github.com/edx/configuration/pull/5223

pomegranited committed:
Updates ansible variables according to new playbook

gr4yscale commented on docs/analytics/insights.md:
```diff
@@ -13,16 +13,10 @@ cd configuration/playbooks
-[`analytics_insights.yml`](resources/playbooks/analytics_insights.yml) and
-[`analytics_api.yml`](resources/playbooks/analytics_api.yml).
-
-If you're planning to expose the `analytics_api` via nginx, you'll also need this
```
Maybe leave a note saying that this template will get installed by running the `analytics_single` playbook?

pomegranited replied:
I don't think we need to leave anything in the docs about this.. there are templates all through the `edx:configuration` code which help us out, and this was only mentioned here because we had to manually add it in.

gr4yscale commented:
@pomegranited 
:+1:

Nice work tracking down the issue with `analytics_api.j2` and suffering through OCIM sandbox issues. Left a quick note about the nginx template, but you know better if it's really useful to mention or not.

- [x] I tested this: I confirmed that both Insights and Analytics API services are accessible on your sandbox, with SSL
- [x] I read through the code
- [ ] I checked for accessibility issues **N/A**
- [x] Includes documentation
- [ ] I made sure any change in configuration variables is reflected in the corresponding
    client's `configuration-secure` repository. **N/A**

gr4yscale commented:
@pomegranited Maybe it's a good idea to mention the mysql user creation issue that you referenced in https://github.com/edx/configuration/pull/5223?

pomegranited committed:
PR review: add note about the mysql issue with analytics_pipeline

Updates the docs and sample vars to make the variables related to the
analytics_single.yml playbook clearer.

pomegranited commented:
Thanks for the review @gr4yscale !  I'm going to leave this unmerged until upstream approves https://github.com/edx/configuration/pull/5223, so our docs match what gets merged.

> Maybe it's a good idea to mention the mysql user creation issue that you referenced in https://github.com/edx/configuration/pull/5223?

Good point -- added with db6ea29
